### PR TITLE
Replaced dirty version of jQuery ui and stopped using jQuery in no-cn…

### PR DESCRIPTION
**Pull request by Gastonius**
…flict mode

### CODE REVIEW

File: chrome_extension/mooltipass-content.js [hunk 1]
```diff
@@ -159,7 +161,10 @@ cipPassword.createDialog = function(inputs, $pwField) {
 	mpJQ("#mooltipass-use-as-password").click(function(e){
 		var password = mpJQ("#mooltipass-password-generator").val();
 
-		mpJQ("input[type='password']:not('.mooltipass-password-do-not-update')").sendkeys( password ).dispatchEvent(new Event('change'));
+		var a = $("input[type='password']:not('.mooltipass-password-do-not-update')");
```
Comment (limpkin) [inline chrome_extension/mooltipass-content.js]: any reason why you're not using mpJQ anymore here?

**Comment by Gastonius** [comment]
Not particularly. But for some reason it wasn't working properly in Safari
(just the sendKeys function)

**Comment by limpkin** [comment]
you mean that using mpJQ would break safari compatibility, or the call to sendKeys would not work because of all the statements attached together?

**Comment by Gastonius** [comment]
The call to sendKeys didn't work until I split it up. If you ask me: makes
no sense at all, still it did behave that way.

**Comment by limpkin** [comment]
I see, for the sake of consistency, could you put back $mpjq there?
Did you make sure we don't get css conflicts as well? I'm not 100% dom isolation is put in place for css...

**Comment by limpkin** [comment]
Moreover, could you make sure that the popup that is brought up when clicking the green icon still works? I remember this one was very problematic when we encountered websites with jquery...

**Comment by Gastonius** [comment]
Yes, a small change tho... do you recall why the background was set to
transparent?  (in the PR)

Tested against those sites: https://wappalyzer.com/applications/jquery-ui

**Comment by limpkin** [comment]
I don't, but I remember we had encountered this problem when testing for conflicts!
So you're 100% we don't have conflicts now? could you change that last mpjq?

**Comment by limpkin** [comment]
@lukas-schulze your input would be greatly appreciated on that PR

**Comment by limpkin** [comment]
but how come are we only noticing this transparent background problem if supposedly we don't have conflicts?

**Comment by Gastonius** [comment]
Old version had "mp-ui-widget-content" set as shown in the image. New version, while it is the same , doesn't contain that. 

screen shot 2017-02-16 at 3 39 52 pm

**Pull request merged**